test/jdk/java/net/httpclient/PlainProxyConnectionTest.java

```diff
@@ -51,18 +53,32 @@
 
 /**
  * @test
- * @bug 8230526
+ * @bug 8230526 8342811
```

Comment:
We only list there bugs for which a change was made under /src and which the test can verify. 
We don't list bugids for which there was only test code changes. In that case we instead add the `noreg-self` label to the JBS issue.

Reply:
Thanks, got it. Removed in 8cec4a7769f71dc0ca44e8c9e0bbfadcb0531927.